lib/page/questions/widget/answer_widget.dart

```diff
@@ -176,6 +181,15 @@ class _AnswerWidgetState extends ConsumerState<AnswerWidget> {
     );
   }
 
+  Widget _buildTextAreaAnswer({
+    required Function onChanged,
```

Comment:
It's better to use [ValueChanged](https://api.flutter.dev/flutter/foundation/ValueChanged.html) type here, what do you think?

Reply:
@markgravity Fixed in https://github.com/chornerman/Survey/pull/101/commits/11ef1528a55d9b45f93c63d242690cc870f5dd97 (to avoid conflict) 🙏🏼